tests/bootstrap.js

```diff
@@ -1,5 +1,5 @@
-const CurlRequest = require('curl-request'),
-      knex = require('knex');
+const knex = require('knex');
+const http = require('http');
```

Comment:
This won't work if the URL is `https` (with the s at the end), and most URLs should be https! There's an `https` module you'll need instead. One of the problems with the native solution is you need to analyse the URL to determine which module to import. So either add that (we have it in some other repos), or use a lib.

Reply:
Yes, I tried `https` module but since our test is using `http` url then it was failing. I'll add a condition to use the right module in case someone changes the test url to https..maybe with regex 👍